src/Button.js

```diff
@@ -112,11 +115,16 @@ Button.propTypes = {
    */
   onClick: PropTypes.func,
   tooltip: PropTypes.string,
-  waves: PropTypes.oneOf(constants.WAVES)
+  waves: PropTypes.oneOf(constants.WAVES),
+  /**
+   * Click-Only
```

Comment:
Maybe we could have a bit more description here as this is what will show on the documentation